react/utils/search.js

```diff
@@ -29,7 +29,29 @@ export const SORT_OPTIONS = [
   },
 ]
 
+// Quick function to get querystring
+// https://stackoverflow.com/a/901144/5313009
+// TODO: Replace this with a better function
+const getParameterByName = (name, url) => {
+  if (!url) url = window && window.location && window.location.href
+  name = name.replace(/[\[\]]/g, '\\$&')
+
+  const regex = new RegExp('[?&]' + name + '(=([^&#]*)|&|#|$)')
+  const results = regex.exec(url)
+
+  if (!results) return null
+  if (!results[2]) return ''
+
+  return window && window.decodeURIComponent && window.decodeURIComponent(results[2].replace(/\+/g, ' '))
+}
+
 export function createInitialMap(params) {
```

Comment:
Couldn't you pass the query here? I mean, where this function is called?

Reply:
I don't think there's much to gain by doing that? This is a hotfix anyway, the very need of this `createInitialMap` needs to be reviewed later

Reply:
Its called in SearchContext.